{"file":"/home/pordefecto/workspace/pruebas-tecncias/transaction-client-test/infrastructure/handler/response/message.go:124","func":"github.com/jpastorm/transaction-client-test/infrastructure/handler/response.API.UnexpectedError","level":"error","msg":"Status: 500:unexpected_error | Endpoint: /api/v1/public/client | Where: github.com/jpastorm/transaction-client-test/infrastructure/handler/response.API.Error:179 | Who: useCase.Create() | UserID: 0 | Err: pq: null value in column \"id\" of relation \"client\" violates not-null constraint","time":"2022-01-10T21:37:25-06:00"}
